crawl-ref/source/mon-cast.cc

```diff
@@ -6189,8 +6189,13 @@ void mons_cast(monster* mons, bolt pbolt, spell_type spell_cast,
         if (sumcount2 <= 0)
             return;
 
+        // Swap chance is 1/2, then 1/3, 1/4, etc. This gives Mara an equal
+        // chance of ending up at any of the original or illusion positions.
```

Comment:
The comment about the odds of swapping belong with the logic, inside `cast_phantom_mirror`. If you have a comment distant from the corresponding logic, then people who change the logic are unlikely to also update the comment.